lib/handler/proxy.c

```diff
@@ -146,6 +146,7 @@ static void on_context_init(h2o_handler_t *_self, h2o_context_t *ctx)
     client_ctx->first_byte_timeout = self->config.first_byte_timeout;
     client_ctx->keepalive_timeout = self->config.keepalive_timeout;
     client_ctx->tunnel_enabled = self->config.tunnel_enabled;
+    client_ctx->force_cleartext_http2 = 1;
```

Comment:
Nice catch.

Would you be interested in changing the assignment to compound form, e.g.,:
```c
*client_ctx = (h2o_httpclient_ctx_t){ ... };
```
so that entries that are not initialized explicitly would be zero-cleared?

Reply:
@kazuho Done in https://github.com/h2o/h2o/pull/2905/commits/9964fa8a9830b20eade43d8f80a7ed47c44033d2

And thank you for giving me the time to realize `force_cleartext_http2 = 1`. That's a terrible mistake.